src/main/java/com/epam/training/gen/ai/repository/ChatHistoryRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.training.gen.ai.repository;
+
+import com.microsoft.semantickernel.services.chatcompletion.ChatHistory;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+@Service
```

Comment:
Minor, I would suggest a more generic stereotype or a more appropriate according to its use. Either @Component or @Repository may give more context. Be careful with @Repository though as it has other features.